front/src/common/redux/set-password/set.password.api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export async function setPassswordApi (request) {
+
+    const url = `${process.env.REACT_APP_SET_PASSWORD_ROUT}${request.data.token}`;
```

Comment:
Please, rename it to `REACT_APP_SET_PASSWORD_ROUT` to `REACT_APP_SET_PASSWORD_ROUTE`